warp10/src/main/java/io/warp10/continuum/gts/GTSPickler.java

```diff
@@ -0,0 +1,102 @@
+//
+//   Copyright 2019  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.continuum.gts;
+
+import com.geoxp.GeoXPLib;
+import net.razorvine.pickle.IObjectPickler;
+import net.razorvine.pickle.PickleException;
+import net.razorvine.pickle.Pickler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Pickler for Geo Time Series
+ */
+public class GTSPickler implements IObjectPickler {
+
+  private static String CLASSNAME_KEY = "classname";
+  private static String LABELS_KEY = "labels";
+  private static String ATTRIBUTES_KEY = "attributes";
+  private static String TIMESTAMPS_KEY = "timestamp";
```

Comment:
timestamp / value / latitude / longitude / elevation should be plurals as they contain arrays of multiple values